dashboard/test/models/regional_partner_organizer_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class RegionalPartnerOrganizerTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
hah, I actually had removed them and just didn't push the commit. Thanks for catching!